Libraries/NativeAnimation/RCTNativeAnimatedNodesManager.h

```diff
@@ -9,10 +9,11 @@
 
 #import <Foundation/Foundation.h>
 
-#import <RCTAnimation/RCTValueAnimatedNode.h>
 #import <React/RCTBridgeModule.h>
 #import <React/RCTUIManager.h>
 
+ #import "RCTValueAnimatedNode.h"
```

Comment:
It was crashing cocoapods build, because only module visible is <React/...>. 

Also it is local dependency and it should be imported like that. The best solution would be completely get rid of this import and replace it by forward declaration of RCTValueAnimatedNodeObserver (which is actually need it in this file). Should I do it?